packages/web/src/components/results/TeeResults.vue

```diff
@@ -1,78 +1,53 @@
 <template>
-
   <!-- DEBUGGING -->
-  <div
-    v-if="debug"
-    class="vue-debug">
+  <div v-if="debug" class="vue-debug">
     <h5>DEBUG - TeeResults</h5>
     <!-- <h6>
       programs.programDetail : <code>{{ programs.programDetail || 'undefined' }}</code>
     </h6> -->
     <div class="fr-grid-row fr-grid-row--gutters fr-mb-3v">
       <div class="fr-col-3">
-        <h6 class="fr-mb-1v"> trackId : <code>{{ trackId }} </code></h6>
+        <h6 class="fr-mb-1v">
+          trackId : <code>{{ trackId }} </code>
+        </h6>
       </div>
       <!-- <div class="fr-col-4">
         <h6 class="fr-mb-1v"> trackConfig : </h6>
           <pre><code>{{ trackConfig }} </code></pre>
       </div> -->
       <div class="fr-col-9">
-        <h6 class="fr-mb-1v"> activeFilters : </h6>
-          <pre><code>{{ activeFilters }} </code></pre>
+        <h6 class="fr-mb-1v">activeFilters :</h6>
+        <pre><code>{{ activeFilters }} </code></pre>
       </div>
     </div>
   </div>
 
   <!-- RESULTS ALERT FOR NO RESULTS BEFORE REFILTERING-->
-  <TeeNoResults
-    v-if="!countFilteredPrograms"
-    :image="trackConfig?.noResultsImage"
-    :message="trackConfig?.noResultsMessage"
-    >
-  </TeeNoResults>
+  <TeeNoResults v-if="!countFilteredPrograms" :image="trackConfig?.noResultsImage" :message="trackConfig?.noResultsMessage"> </TeeNoResults>
 
   <!-- RESULTS CALLBACK -->
-  <h4
-    v-if="countFilteredPrograms && trackConfig?.showResultsTitle"
-    class="fr-pt-12v">
+  <h4 v-if="countFilteredPrograms && trackConfig?.showResultsTitle" class="fr-pt-12v">
     {{ choices.t('results.fittingPrograms') }}
     ({{ countFilteredPrograms }})
   </h4>
 
   <!-- PROGRAMS AS LIST OF CARDS -->
-  <div
-    v-if="countFilteredPrograms"
-    class="fr-container fr-px-0 fr-mt-6v">
-
+  <div v-if="countFilteredPrograms" class="fr-container fr-px-0 fr-mt-6v">
     <!-- RESULTS SIZE -->
-    <div
-      v-if="countFilteredPrograms > 1"
-      class="fr-mb-4v tee-text-light">
+    <div v-if="countFilteredPrograms > 1" class="fr-mb-4v tee-text-light">
       {{ countReFilteredPrograms }}
       {{ choices.t('results.results') }}
     </div>
 
     <!-- FILTERS IF ANY -->
-    <div
-      v-if="trackConfig?.filters && countFilteredPrograms > 1"
-      class="fr-grid-row fr-grid-row--gutters fr-mb-4v">
-      <div
-        v-for="filter in trackConfig.filters"
-        :key="filter.label"
-        class="fr-col">
-        <TeeResultsFilter
-          :filter="filter"
-          :debug="debug"
-          @updateFilter="updateFilters"/>
+    <div v-if="trackConfig?.filters && countFilteredPrograms > 1" class="fr-grid-row fr-grid-row--gutters fr-mb-4v">
+      <div v-for="filter in trackConfig.filters" :key="filter.label" class="fr-col">
+        <TeeResultsFilter :filter="filter" :debug="debug" @updateFilter="updateFilters" />
```

Comment:
issue : `v-on event '@updateFilter' must be hyphenated`

Reply:
@yvalentin je sais pas si t'avais vu cette remarque. C'est assez minime et ça me semble être cohérent avec d'autres changements vus dans des fichiers de composants